Kitodo/src/main/java/org/kitodo/production/forms/BaseForm.java

```diff
@@ -62,6 +62,7 @@
 
     protected static final String REDIRECT_PATH = TEMPLATE_ROOT + "{0}?" + REDIRECT_PARAMETER;
     protected static final String DEFAULT_LINK = "desktop";
+    protected static final String DESKTOP_LINK = "desktop.jsf";
```

Comment:
Instead of adding a new constant that basically just adds the `.jsf` suffix to the value of `DEFAULT_LINK` in the prior line, it would be less redundant to just add that suffix to the existing constant and reuse it. `DEFAULT_LINK` is currently only used for the `referrer` URL parameter in some places, and for that purpose `desktop.jsf` works as well as just `desktop`.